### DISCUSSION TITLE
Why do moving objects duplicate in recordings?

### DISCUSSION THREAD

**Discussion by husain3** [Motion-Project/motion#1557] [Q&A]
<img width="1009" alt="Screen Shot 2022-08-01 at 11 38 54 AM" src="https://user-images.githubusercontent.com/12962124/182209595-8c0fd8b2-c9f3-453a-ba28-7527465d145c.png">

Even when I use movie_passthrough (all rtsp streams are h264) I get these decompression artifacts. My processor is an Intel i5-6500T. Might it be because I'm writing these to a USB 3.0 stick?

These past months I've tried many configurations to try and fix it. In the year and a half I've been using Motion, it started showing up about 6 months ago. What could be the issue?

**Comment by Mr-Dave** [comment]
Some things to try.  Specify a higher ```capture_rate``` for Motion, specify a lower FPS inside of the camera configuration, specify a lower number for I frame rate inside of the camera, change from a wifi connection to a hardwire connnection.

What is happening is that the camera is compressing images and then sending them.  What is arriving at Motion isn't enough information to un-compress them.  So to get the complete sequence of compressed information from the camera, either specify Motion to "speed up" by increasing the ```capture_rate``` or get into the camera configuration and tell it to "slow down" sending the pictures by lowering the camera specified FPS....or tell the camera to not compress the data so much by specifying a lower number for "I frames".

**Reply by Mr-Dave** [reply]
Or....it could definitely be that the writing to the USB slows everything down and breaks it all because everything is waiting on the USB to finish its thing.....

**Reply by husain3** [reply]
I tried to increase the `capture_rate` to no avail. I don't have access to direct camera settings to lower the fps. I think it should be noted that when I use Frigate, I don't get these decompression artifacts. I reckon it's because it freezes the video and resumes once it has enough data for decompression?

**Reply by husain3** [reply]
I think it should also be noted that I started using Motion around April 2021 but only started getting these decompression artifact errors a few months ago. Has there been any changes recently in that area of the code?

**Reply by husain3** [reply]
However I am looking at the logs and there seems to be this error: 

`[0:motion] [ALR] [ALL] conf_cmdparse: Unknown config option "capture_rate"`

**Reply by husain3** [reply]
Is there maybe something else? `rtsp_uses_tcp`?
